src/main/java/org/sonar/plugins/stash/issue/collector/StashCollector.java

```diff
@@ -41,7 +41,17 @@ public static StashCommentReport extractComments(String jsonBody) throws StashRe
           // can be null if comment is attached to the global file
           Long line = (Long) jsonAnchor.get("line");
           
-          StashComment comment = new StashComment(id, message, path, line);
+          long version = (long) jsonComment.get("version");
+          
+          JSONObject jsonAuthor = (JSONObject) jsonComment.get("author");

```

Comment:
doesn't it make sense to use extractUser()?
https://developer.atlassian.com/static/rest/stash/3.11.3/stash-rest.html#idp1467264
emailAddress insteadof email, no?